### ISSUE TITLE
Getting message `Sorry, an unexpected error occurred. Quitting and restarting may help...`

### ISSUE THREAD

**Issue by TusharSheth** [Jigsaw-Code/outline-client#782]
**Describe the bug**
I am not able to connect to server. Getting error message.

- getting error at file `C:\Program Files (x86)\Outline\resources\app.asar\node_modules\@sentry\browser\dist\index.js`

`    // this fails for some browsers. :(`
`         if (originalConsoleLevel) {`
`         Function.prototype.apply.call(originalConsoleLevel, global$2.console, args);`
`         }`

**Screenshots**
![outline](https://user-images.githubusercontent.com/19387476/80300278-df66a600-87b8-11ea-8759-eedaa976cc49.PNG)

**Desktop (please complete the following information):**
 - OS: windows 8.1 pro
 - Browser: Chrome
 - Version:

**Comment by zakariashahedbappa** [comment]
same problem to me also

**Comment by iamsid47** [comment]
If this is failing on Chrome, you could try using another browser. Or, it can also happen that some of the code which you have written in the index.js file may be invalid. You should perhaps, focus on the syntax.

PS. Can you tell me in detail about what have you actually done/edited.

**Comment by TusharSheth** [comment]
@iamsid47 This screenshot is taken by enabling developer tool, not from any browser. 

Let me tell you, If you haven't used Outline, it is developed in Electron. 

Plus I haven't changed any code.

**Comment by iamsid47** [comment]
Okay, so in that case can you tell me that do you get this error when you start the application or the time when you try to connect it to your server?

Yeah! I know that it is developed in Electron for Windows/Linux/MAC and Cordova for android in addition with Polymer :)

**Comment by TusharSheth** [comment]
Error is getting occurred while connecting to server. You need to consider the Windows version: 8.1 Pro. Working fine on Windows 10 Pro.

**Comment by iamsid47** [comment]
Well, in that case you should consider contacting the official contributers and they'll surely help you get this issue solved up :)

**Comment by TusharSheth** [comment]
> Well, in that case you should consider contacting the official contributers and they'll surely help you get this issue solved up :)

Yeah I know. That's why I have reported an issue on their official repository.  🤞

**Comment by iamsid47** [comment]
I mean you should consider contacting the developers directly.

**Comment by kendytuan** [comment]
@TusharSheth Hi bro, i and some my friends have same your problem while installing lastest version that downloaded from official website.
OS: Windows 7, 8.1 and 10

I tried old version from 2019 and it's worked. 
So, i think the new version has something missing.

Try my installation file bro!!

> https://drive.google.com/file/d/1ObZyHsB_k59kFe060p8UMfBcRCw3H-MP/view

**Comment by ChrainWang** [comment]
Hi, I got the same error message on win10 and I did solved it.  It's caused by missing MSVCP140.dll and VCRUNTIME140.dll on my PC.  Installing vcredist solved my problem. You can find it here https://support.microsoft.com/en-us/help/2977003/the-latest-supported-visual-c-downloads. I installed both x64 and x86 version, but I guess only x86 is needed to make it work.

If you are not sure whether it is the same problem, you may try running c:\Program Files(x86)\Outline\smartdnsblock.exe in command prompt. 

Also you can try running c:\Program Files(x86)\Outline\outline.exe in command prompt to get the error log. It may help trouble shooting.

**Comment by joeytwiddle** [comment]
Thanks ChrainWang, installing the DLLs did help!

And you are right, just the **x86 DLLs** were needed, to match x86 Outline.

**Comment by UnKnoWn-Consortium** [comment]
I am convinced this x86 VC redistributable dependency specific to the Windows platform should be mentioned somewhere in the README.md, apart from making the error message more informative...
